persistence-model/src/main/resources/application-dev.conf

```diff
@@ -27,6 +29,17 @@ akka {
       "hmda.persistence.messages.events.processing.HmdaFileParserEvents$LarParsedErrors" = parser
       "hmda.persistence.messages.events.validation.SubmissionLarStatsEvents$SubmittedLarsUpdated" = larStats
       "hmda.persistence.messages.events.validation.SubmissionLarStatsEvents$MacroStatsUpdated" = larStats
+      "hmda.persistence.messages.events.processing.HmdaFileValidatorEvents$TsSyntacticalError" = validator
```

Comment:
Adding the `HmdaFileValidator` events in this PR because I forgot to add them in #988 😑